src/implementations/twig/components/table/__snapshots__/table.test.js.snap

```diff
@@ -744,8 +742,6 @@ exports[`Table Default renders correctly with row extra attributes 1`] = `
         </tr>
         <tr
           class="ecl-table__row"
-          data-test=""
```

Comment:
extra attributes are no longer tested here

Reply:
wow, this looked weird, since nothing was touched in the template or the data..but:
it turned out that the current template is actually expecting an impossible data structure, the rows array contains an array of objects with the cell data, we did expect an extra_class and extra_attributes props with a value, which is not valid json or js, it was working because we were adding props to these objects dynamically in the test, but you could not reproduce that data structure. 
This is the reason why with the new twing, these attributes and classes where not rendered anymore.
Furthermore the extra_attributes as a string cannot be supported on our side, they would be escaped and we cannot use |raw, so i've used for those the same approach we have for the main attributes.
The overall result is that now we would have the row cells inside a "data" named object, and then we have extra_classes as a string and extra_attributes as an array of objects